models/spike_detector.cpp

```diff
@@ -45,7 +45,7 @@ nest::spike_detector::spike_detector()
   : Node()
   // record time and gid
   , device_( *this, RecordingDevice::SPIKE_DETECTOR, "gdf", true, true )
-  , user_set_precise_times_( false )
+  , has_user_set_precise_times_( false )
```

Comment:
I don't agree with this name change, I think the original name is sufficiently clear (note that "set" is past tense in the original name).